src/Microsoft.AspNetCore.Razor.Tasks/DotnetToolTask.cs

```diff
@@ -29,6 +34,10 @@ public abstract class DotNetToolTask : ToolTask
         // Specifies whether we should fallback to in-process execution if server execution fails.
         public bool ForceServer { get; set; }
 
+        // Specifies whether server execution is allowed when PipeOptions.CurrentUserOnly is not available.
+        // For testing purposes only.
+        public bool AllowInsecureServer { get; set; }
```

Comment:
This is something temporary so we can still run our tests on the build server. This can go away once we start running our tests on netcoreapp2.1.

Reply:
Also name suggestions are welcome.

Reply:
I would avoid the word insecure. I would say something like `SuppressCurrentUserOnlyPipeOptions`